packages/editor/templates/Coin/contracts/Coin.sol

```diff
@@ -1,35 +1,56 @@
+// Specifies that the source code is for a version
+// of Solidity greater than 0.5.10
```

Comment:
Isn't this specifying greater than or equal to`0.5.0`, not greater than `0.5.10`?

Reply:
I have a vague feeling that someone somewhere told me this isn't the case, but it _feels_ like you're right and I don't know why this  is like this…

```suggestion
// of Solidity greater than 0.5.0
```

Reply:
Well unless you can cite a source, I'm fairly positive `pragma solidity >=0.5.0` means greater than or equal to `0.5.0`, not greater than `0.5.10`...